pkg/lib/theme/logo.go

```diff
@@ -0,0 +1,104 @@
+package theme
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/tdewolff/parse/v2"
+	"github.com/tdewolff/parse/v2/css"
+)
+
+// MigrateCreateCSSWithDefaultLogoHeight create a css file with default logo height for existing projects that does not have theme customization at all
+func MigrateCreateCSSWithDefaultLogoHeight(selector string) (result []byte, err error) {
```

Comment:
Remove this function

Reply:
I understand `SetDeclarationInSelector` can replace `MigrateSetDefaultHeight`; but how is `SetDeclarationInSelector` supposed to replace `MigrateCreateCSS..` from scratch?

Is caller expected to

```diff
--- MigrateCreateCSS(":root")
+++ SetDeclarationInSelector(":root{}", ":root", Declaration{...})
```

Reply:
You can make Ruleset and indentation public and write

```
newCSS := RuleSet{
  Selector: ":root",
  Declarations: []*Declaration{
    &declaration,
  },
}
// And then stringify it.
```